tests/components/motion_blinds/test_config_flow.py

```diff
@@ -56,6 +143,51 @@ async def test_config_flow_manual_host_success(hass):
     }
 
 
+async def test_config_flow_discovery_2_success(hass):
+    """Successful flow with 2 gateway discovered."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+
+    assert result["type"] == "form"
+    assert result["step_id"] == "user"
+    assert result["errors"] == {}
+
+    with patch(
+        "homeassistant.components.motion_blinds.config_flow.MotionDiscovery.discover",
+        return_value=TEST_DISCOVERY_2,
+    ):
+        result = await hass.config_entries.flow.async_configure(
+            result["flow_id"],
+            {},
+        )
+
+    assert result["type"] == "form"
+    assert result["step_id"] == "select"
+    assert result["errors"] == {}
```

Comment:
Optional: I could see adding a test here for verifying the hosts in the input select, but that said I found it a bit hard to test detailed of data schemas.

Reply:
yea, I agree that would be nice, but I would not know how to do that.
Writing tests is not my strongest point.

Reply:
I figured it out and included a test to check the hosts in the input select:
`    assert result["data_schema"].schema['select_ip'].container == [TEST_HOST, TEST_HOST2]`